src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -336,18 +336,39 @@ internal virtual LexicalSortKey GetLexicalSortKey()
 
 #nullable enable
 
-        public virtual Location? TryGetFirstLocation()
```

Comment:
Not sure what is the intent here, but I do not think this API should be removed. In any PR.